i18n/fr.yaml

```diff
@@ -12,6 +12,9 @@
 - id: meta_lastmod
   translation: "Dernière modification"
 
+- id: meta_translations
```

Comment:
@pkel, @jpguerard,
could you please verify this translation?

Reply:
Seems fine to me.

Reply:
OK, thank you.